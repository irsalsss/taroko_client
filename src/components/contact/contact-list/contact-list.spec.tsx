import { render } from "@/utils/test/react-testing-setup";
import { wrapperReactQuery } from "@/utils/test/wrapper-testing";
import ContactList from "./contact-list";

describe("ContactList", () => {
  it("should render ContactList successfully", () => {
    const { baseElement } = render(<ContactList />, {
      wrapper: wrapperReactQuery,
    });

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
