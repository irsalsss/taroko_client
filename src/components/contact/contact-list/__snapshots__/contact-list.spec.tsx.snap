// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactList should render ContactList successfully 1`] = `
<body>
  <div>
    <div
      class="flex justify-center items-center p-8 mt-8"
    >
      <h5>
        The 
        contacts
         
         are empty...
      </h5>
    </div>
    <div
      class="flex flex-wrap gap-4 px-4 mt-4"
    />
  </div>
</body>
`;
